test/support/pages/settings_page.js

```diff
@@ -14,16 +14,20 @@ export default (function () {
       return this.remote.findDisplayedByCssSelector('.app-link:nth-child(5) a').click();
     },
 
-    clickPath: function (path) {
-      return this.remote.findDisplayedByCssSelector('[kbn-href="#/management/' + path + '"]').click();
+    clickLinkText: function (text) {
+      return this.remote.findDisplayedByLinkText(text).click();
```

Comment:
I switched to using the linked text because some paths have trailing slashes and it overall made the interface more predictable.